tests/tests/swfs/avm2/mouse_over_while_dragging/Test.as

```diff
@@ -0,0 +1,61 @@
+package {
+    import flash.display.MovieClip;
+    import flash.display.Sprite;
+    import flash.events.Event;
+    import flash.events.MouseEvent;
+    import flash.system.fscommand;
+
+    public class Test extends MovieClip {
+        private var dragger:MovieClip;
+        private var target:MovieClip;
+        private var sawOver:Boolean = false;
+
+        public function Test() {
+            addEventListener(Event.ADDED_TO_STAGE, onAdded);
+        }
+
+        private function onAdded(_:Event):void {
+            removeEventListener(Event.ADDED_TO_STAGE, onAdded);
+
+            dragger = new MovieClip();
+            dragger.graphics.beginFill(0xCC3333);
+            dragger.graphics.drawRect(0, 0, 100, 100);
+            dragger.graphics.endFill();
+            dragger.x = 0;
+            dragger.y = 0;
+            dragger.buttonMode = true;
+            addChild(dragger);
+
+            target = new MovieClip();
+            target.graphics.beginFill(0x33CC66);
+            target.graphics.drawRect(0, 0, 120, 120);
+            target.graphics.endFill();
+            target.x = 200;
+            target.y = 200;
+            addChild(target);
+
+            target.buttonMode = true;
+            target.mouseChildren = false;
+
+            target.addEventListener(MouseEvent.MOUSE_OVER, function(e:MouseEvent):void {
+                trace("OVER buttonDown=" + e.buttonDown);
+                sawOver = true;
+            });
+            target.addEventListener(MouseEvent.MOUSE_OUT, function(e:MouseEvent):void {
+                trace("OUT buttonDown=" + e.buttonDown);
+                if (sawOver) {
+                    fscommand("quit");
```

Comment:
This causes the last line not to be traced. Why is this here? And most importantly, how does the test pass in Ruffle? :D

Reply:
I'm not confident about the behavior, but seems we run 1 tick and then running all events. So, we get all events which ran within 1 tick.

Reply:
I think you're right, the test runner has a bug and doesn't consider fscommands between events.